anthem/lyrics/settings.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Camptocamp SA
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0.en.html)
+
+
+def define_settings(ctx, model, values):
+    """ Define settings like being in the interface
+     Example :
+      - model = 'sale.config.settings'
+      - values = {'default_invoice_policy': 'delivery'}
+     Be careful, settings onchange are not triggered with this function.
+    """
+    ctx.env[model].create(values).execute()
```

Comment:
the model argument should accept a instance too, so we can change its context such as:

```
define_settings(ctx, ctx.env['sale.config.settings'].with_context(company_id=1), {...})
```

```